nixos/modules/programs/wayland/wayland-session.nix

```diff
@@ -1,4 +1,9 @@
-{ lib, pkgs, xwayland ? true }:
+{
+  lib,
+  pkgs,
+  xwayland ? true,
+  wlr-portal ? true,
```

Comment:
Shouldn't those be prefixed with enable?

Reply:
Sure.

Reply:
Done in https://github.com/NixOS/nixpkgs/pull/315827/commits/c6c6cc952561936e4abb618ef35302c0bf1c96ad